src/ex5/server/db/models/item.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+const {
+  Model
+} = require('sequelize');
+module.exports = (sequelize, DataTypes) => {
+  class Item extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+    }
+  }
+  Item.init({
+    itemId: DataTypes.STRING,
+    itemName: DataTypes.STRING,
+    isPokemon: DataTypes.BOOLEAN,
+    pokemonId: DataTypes.INTEGER,
+    pokemonData: DataTypes.STRING,
```

Comment:
I don't think you need all 3 columns. isPokemon can be removed if you have pokemonId, as other items will have pokemonId == null.
it can also be removed and just saved as an object in the pokemonData..
you need to remember that DB has a price in the real world and also that you don't want to have 2 sources of truth.